File: Leetcode/typescript/1684-count-consistent-strings.ts
function countConsistentStrings(allowed: string, words: string[]): number {
  const allowedSet = new Set(allowed);
  let res = 0;

  for (const word of words) {
    if ([...word].every((char) => allowedSet.has(char))) {
      res++;
    }
  }
  return res;
}

File: Leetcode/python/1684-count-consistent-strings.py
from typing import List


class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        res = 0
        allowedSet = set(allowed)

        for word in words:
            if all(char in allowedSet for char in word):
                res += 1

        return res
